### PR TITLE
COMMERCE-1398 Fix the groupId we're using for subscription entries

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

File: commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceSubscriptionEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -118,18 +118,14 @@ public CommerceSubscriptionEntry addCommerceSubscriptionEntry(
 
 		validateCPSubscriptionType(cpSubscriptionType);
 
-		long groupId =
```
Comment (marco-leo) [inline commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceSubscriptionEntryLocalServiceImpl.java:121]: Add groupId as parameter

**Comment by alecsloan** [comment]
@marco-leo requested changes made, please review.

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:stop

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:stop

**Comment by alecsloan** [comment]
ci:test

**Pull request merged**